src/System.Net.Security/ref/4.0.0/System.Net.Security.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
You don't need this ref/4.0.0 folder. We will get this asset from the last stable package.